content/dstu2.md

```diff
@@ -53,24 +53,35 @@ Cerner supports the FHIR<sup>®</sup> standard defined media types for JSON cont
 
 We encourage you to explicitly request one of these types via the `Accept` header.
 
-## Root Endpoint
+## Service Root URL
 
-The URL takes the form `fhir.sandboxcernerpowerchart.com/dstu2/:ehr_source_id/:resource`.
+URLs for the FHIR server vary by the tenant (datasource or client) being accessed, as well as other factors. If the 
+application is a SMART application, the [service root url](http://hl7.org/fhir/dstu2/http.html#general) is provided at 
+launch time. For standalone applications, the URL can be requested (or configured) when the application is set up to run
+ against a specific tenant. FHIR calls will be made against URLs of the following format:
 
-### Open Endpoint
+`:serviceRootURL/:resource\[?:parameters]`
```

Comment:
It did before I put it in quotes (at least, to make it not report issues or highlight as a link in the ide). Removed :)